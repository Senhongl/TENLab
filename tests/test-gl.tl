a = 1;
b = 1;
c = 2;
def foo(a) {
    a = 2;
    print(b);
    b = c;
    print(b);
}

foo(a);
print(a);
print(b);