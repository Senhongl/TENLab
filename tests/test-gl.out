1
[ CPUIntType{} ]
2
[ CPUIntType{} ]
1
[ CPUIntType{} ]
1
[ CPUIntType{} ]
